package.json

```diff
@@ -60,51 +60,19 @@
 		"vuex-router-sync": "^5.0.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.5",
-		"@babel/eslint-parser": "7.14.5",
 		"@babel/plugin-proposal-object-rest-spread": "^7.14.5",
-		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
-		"@babel/preset-env": "^7.14.5",
+		"@nextcloud/babel-config": "^1.0.0-beta.1",
 		"@nextcloud/browserslist-config": "^2.1.0",
 		"@nextcloud/eslint-config": "^5.1.0",
-		"@nextcloud/eslint-plugin": "^2.0.0",
+		"@nextcloud/stylelint-config": "^1.0.0-beta.0",
 		"@nextcloud/webpack-vue-config": "^4.0.3",
 		"babel-core": "^7.0.0-bridge.0",
```

Comment:
Should not be needed

Reply:
Ah, for jest?

Reply:
Yes, this is for jest. Although I didn't test it without.